src/main/java/cooper/parser/SignInDetailsParser.java

```diff
@@ -71,6 +71,8 @@ private SignInProtocol parseSignInDetails(String input) throws UnrecognisedComma
             var result = optResult.get();
             String command = result.allCommands().get(0).name();
             List<Argument> commandArgs = result.allCommands().get(0).arguments();
+            System.out.println(command);
+            System.out.println(commandArgs);
```

Comment:
Remember to remove the lines you used for debugging 🤣

Reply:
Opps

Reply:
Oh no, why are there conflicts now? 😨

Reply:
I deleted the files under tmp folder because now there are two separate files for signin parser and command parser each. Somehow this was marked as conflict. I have resolved it just now.